go-runtime/ftl/ftltest/fake.go

```diff
@@ -64,13 +68,14 @@ type fakeFTL struct {
 // type but is not constrained by input/output type like ftl.Map.
 type mapImpl func(context.Context) (any, error)
 
-func contextWithFakeFTL(ctx context.Context) context.Context {
+func contextWithFakeFTL(ctx context.Context, options []Option) context.Context {
```

Comment:
Could also use the common pattern of `options ...Option`, so the caller doesn't necessarily have to construct a list or even add a new argument.